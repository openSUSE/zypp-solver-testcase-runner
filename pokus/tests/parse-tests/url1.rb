#!/usr/bin/ruby


# Specify a broken URL.


require '../../src/testsuite'


Testsuite.set_arch("x86_64")

begin
    pool = Testsuite.read_repo("htp://site/repo/")
rescue ZYppException => e
    puts "ZYppException caught"
    puts e.to_s
else
    puts "Oh, no exception caught"
    exit 1
end

