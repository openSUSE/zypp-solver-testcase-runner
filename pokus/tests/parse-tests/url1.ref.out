ZYppException caught
History:
 - Unsupported URL scheme in URL: htp://site/repo/
Error trying to read from htp://site/repo/
